web/cobrands/oxfordshire/layout.scss

```diff
@@ -37,24 +37,14 @@ body, body a {
   display: none;
 }
 
-// White background, so no shadow or margin needed.
+// White background, so no margin needed.
 .content {
     margin: 0;
-    @include box-shadow(none);
-}
-.iel8 {
-    .content {
-        margin: 0;
```

Comment:
This margin should presumably remain to override the` .iel8 .content` margin from sass.